napari/_qt/event_loop.py

```diff
@@ -6,6 +6,25 @@
 from qtpy.QtGui import QPixmap
 from qtpy.QtWidgets import QApplication, QSplashScreen
 
+from .qt_event_timing import QApplicationWithTiming
+from ..utils import perf
+
+
+def _create_application(argv) -> QApplication:
+    """Create our QApplication.
+
+    Notes
+    -----
+
+    We substitute QApplicationWithTiming when using perfmon. Note that we also
+    call convert_app_for_timing() in Viewer but that one is for Jupyter/IPython
+    while this one is for running from the command line.
+    """
+    if perf.USE_PERFMON:
```

Comment:
might we want to just directly use `os.getenv` here and put the `from .qt_event_timing` import below the conditional, to avoid importing everything in the perf module when we aren't using it?
(I looked through, and it seems to be small/fast imports... but even still?)

Reply:
Yes fixed this so perf is imported lazy. I suspect the import time is negligible but it seems reasonable to keep perf totally lazy.